src/App.js

```diff
@@ -36,14 +39,35 @@ class App extends React.Component {
       // .query
       const response = await axios.get(url);
       console.log('response Object: ', response);
+      console.log(this.state.searchQuery);
       console.log(response.data[0].lat, response.data[0].lon);
-      this.setState({ location: response.data[0].display_name, lat: response.data[0].lat, lon: response.data[0].lon, errorMessage: false });
+      this.setState({ location: response.data[0].display_name, lat: response.data[0].lat, lon: response.data[0].lon, errorMessage: false }, ()=> this.getWeather() );
```

Comment:
While you're troubleshooting your code it would likely be helpful to break out your setState values onto new lines so each value is displayed on its own line, and help improve readability to easier identify bugs.